tests/lib/eslint.js

```diff
@@ -974,23 +974,6 @@ describe("eslint", function() {
             eslint.verify(code, {}, filename, true);
         });
 
-        it("should mark variables in Node.js environment as used", function() {
```

Comment:
I had commented it out in my first commit, and then mindlessly deleted it because it was commented out. I'll be adding it back in.